<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SubmitMain</class>
 <widget class="QMainWindow" name="SubmitMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>商品发布</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="titleLabel">
          <property name="text">
           <string>商品标题</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="detailLabel">
          <property name="text">
           <string>商品详情</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="manLabel">
          <property name="text">
           <string>联系人</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="contactLabel">
          <property name="text">
           <string>联系方式</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="titleLineEdit"/>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="ManLineEdit"/>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="contactLineEdit"/>
        </item>
        <item row="1" column="1">
         <widget class="QPlainTextEdit" name="detailTextEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="acceptBtn">
        <property name="styleSheet">
         <string notr="true">color: white;
background-color: #48c9b0;
border-color: #48c9b0;
font-size: 17px;
border-radius: 6px;
font-weight: normal;
border: none;
border: 2px solid #bdc3c7;
display: block;
width: 100%;
float: left;
margin: 2px 0 2px;
font: 9pt &quot;黑体&quot;;
font-size: 11px;
padding-top: 5px;
padding-bottom: 5px;</string>
        </property>
        <property name="text">
         <string>确认</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelBtn">
        <property name="styleSheet">
         <string notr="true">color: white;
background-color: #48c9b0;
border-color: #48c9b0;
font-size: 17px;
border-radius: 6px;
font-weight: normal;
border: none;
border: 2px solid #bdc3c7;
display: block;
width: 100%;
float: left;
margin: 2px 0 2px;
font: 9pt &quot;黑体&quot;;
font-size: 11px;
padding-top: 5px;
padding-bottom: 5px;</string>
        </property>
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
